packages-exp/auth-exp/test/mock_auth.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AppName, ApiKey, Auth } from '../src/model/auth';
+
+export function mockAuth(name: AppName, apiKey: ApiKey): Auth {
```

Comment:
I have something similar in my api_helper branch...

Reply:
I'll wait for yours to merge and use it